Unify Formatting (#82)
C2Devel/terraform-provider-croccloud: website/docs/d/eks_node_groups.md

layout: "aws"
page_title: "aws_eks_node_groups"
description: |-
  Provides information about the EKS node groups names associated with a named EKS cluster.
---

# Data Source: aws_eks_node_groups

> Provides information about the EKS node groups names associated with a named EKS cluster.

Reviewer: ```suggestion
Provides a list of EKS node groups names associated with an EKS cluster.
```

Нужно продублировать в служебный блок.
После описания удалилась пустая строка перед заголовком `Example Usage`, верни пожалуйста

* `cluster_name` - (Required) The name of the cluster.

> ## Attribute Reference

Reviewer: ```suggestion
## Attribute Reference

In addition to all arguments above, the following attributes are exported:
```
